tapgarden/caretaker.go

```diff
@@ -195,8 +198,17 @@ func (b *BatchCaretaker) Cancel() error {
 		if err != nil {
 			err = fmt.Errorf("BatchCaretaker(%x), batch state(%v), "+
 				"cancel failed: %w", batchKey, batchState, err)
+
+			b.cfg.PublishMintEvent(newAssetMintErrorEvent(
+				cancelResp.err, BatchStateSeedlingCancelled,
```

Comment:
P sure this should just be `err`, as `cancelResp` is only populated at the end of this scope?

Reply:
Oops, left over from previous iteration. You're right of course, fixed. Same below.